client/components/TrackImage.jsx

```diff
@@ -8,4 +9,19 @@ const TrackImage = style.div`
   background-size: 50px 50px;
 `;
 
-export default TrackImage;
+
+export default class PlayTrackImage extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    return (
+      <TrackImage src={this.props.src} style={{ float: "left", marginRight: ".5em", }}>
+        <PlayTracksIcon display={this.props.isSelected ? "inline-block" : "none"} style={{ margin: "auto", width: "50px", height: "50px" }} />
```

Comment:
here as well display the component conditionally instead of using css

Reply:
I will resolve this change in the next version due to a css sizing issue that was blocking me. I will ask follow up in person on Tuesday.